app/src/main/java/com/asfoundation/wallet/analytics/AmplitudeAnalytics.kt

```diff
@@ -0,0 +1,66 @@
+package com.asfoundation.wallet.analytics
+
+import android.content.Context
+import com.amplitude.api.Amplitude
+import com.amplitude.api.Identify
+import com.amplitude.api.TrackingOptions
+import com.asf.wallet.BuildConfig
+import com.asfoundation.wallet.identification.IdsRepository
+import com.google.android.gms.common.ConnectionResult
+import com.google.android.gms.common.GoogleApiAvailability
+import io.reactivex.schedulers.Schedulers
+
+class AmplitudeAnalytics(private val context: Context, private val idsRepository: IdsRepository) :
+    AnalyticsSetUp {
+
+  private val amplitudeClient = Amplitude.getInstance()
+  private lateinit var entryPoint: String
+
+  override fun setUserId(walletAddress: String) {
+    amplitudeClient.userId = walletAddress
+  }
+
+  override fun setGamificationLevel(level: Int) {
+    val identify = Identify().append(AmplitudeEventLogger.USER_LEVEL, level)
+        .append(AmplitudeEventLogger.APTOIDE_PACKAGE,
+            BuildConfig.APPLICATION_ID)
+        .append(AmplitudeEventLogger.VERSION_CODE, BuildConfig.VERSION_CODE)
+        .append(AmplitudeEventLogger.ENTRY_POINT,
+            if (entryPoint.isEmpty()) "other" else entryPoint)
+        .append(AmplitudeEventLogger.HAS_GMS, hasGms())
```

Comment:
Improve codestyle by keeping everything in the same line (yes Kotlin codestyle is bad)